modules/config.py

```diff
@@ -420,12 +420,13 @@ def update_libraries(self, test):
             logger.info("")
             util.seperator("{} Library {}Collections".format(library.name, "Test " if test else ""))
             collections = library.collections
-            if collections:
+            collections_to_process = (collections.keys() & requested_collections.split(",")) if requested_collections else collections
```

Comment:
You can use my util.get_list method to make sure the command line arg will work with "," or ", ".
```python
collections_to_process = (collections.keys() & util.get_list(requested_collections)) if requested_collections else collections
```